### DISCUSSION TITLE
Not show site_name:  on index

### DISCUSSION THREAD

**Discussion by carlosrcoelho** [squidfunk/mkdocs-material#5532] [Q&A]
Hi!

I've been having a hard time with this. I want just the site name to be shown on the top and nothing else in the index. If you see this image, I put a name on top, the same name is repeated below, however, I don't want this to happen.

![names](https://github.com/squidfunk/mkdocs-material/assets/80289056/1e6cd789-7ae2-46df-9b56-7d99bbedf7de)

**Comment by santaimpersonator** [comment]
Try:
* Defining the [`site_name`](https://www.mkdocs.org/user-guide/configuration/#site_name)
* Don't enable the [tab navigation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=nav#without)
* [Hide the navigation and toc sidebars](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/?h=nav#hide-both) on the page
